src/Symfony/Component/Lock/Store/ZookeeperStore.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Store;
+
+use Symfony\Component\Lock\Exception\LockAcquiringException;
+use Symfony\Component\Lock\Exception\LockConflictedException;
+use Symfony\Component\Lock\Exception\LockReleasingException;
+use Symfony\Component\Lock\Exception\NotSupportedException;
+use Symfony\Component\Lock\Key;
+use Symfony\Component\Lock\StoreInterface;
+use Zookeeper;
+use Throwable;
```

Comment:
Symfony does not "import" classes without namespace. Use directly `\Throwable` when needed

Reply:
Do you mean they don't get autoloaded without namespace? These should be in global namespace and it works fine when I test locally. I don't know if it's a standard but I prefer declaring them at top so I don't have to add `\` for every usage. If I am missing something can you please elaborate on this? Thanks!

Reply:
In PHP you have two options when working with namespace:
1. importing the namespace (what you have done)
2. using it directly (ex: new \Foo\Bar)

In Symfony, we never import global namespace (like Throwable, DateTime, SplFileObject, Exception ...)

So you have to update your code to remove the `use Throwable`, then use directly `\Throwable`

Thanks